client/src/components/behavior/chart/BehavioralChart.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { ResponsiveContainer, Line, LineChart, Tooltip, Legend, YAxis, XAxis, CartesianGrid } from 'recharts';
+
+const BehavioralChart = props => {
+  const { observations } = props;
+
+  const behaveData = observations.slice(observations.length - 7, observations.length).map(obj => {
```

Comment:
We shouldn't `slice` here.

Reply:
So the idea was to have the last 7 days record show up. Is there a more adequate way to do it?

Reply:
Hmm okay. I haven't looked at the `rechart` docs, but is there a way to have some range on the x-axis?

Reply:
I can look into that. I am sure there is. 
If there is a range and let's say X-axis is limited to 7, the goal would be to show only the last 7 days. So we will have to limit the 7 records going there from the map. OR make sure the rechart only takes in the data from the last 7 entries. Also if this is not necessary feature at the moment then we can also integrate it in the next iteration. The idea is to not have the chart crowded (hence the 7 days limit).

Reply:
Yeah, this would be added in the next iteration.

Currently, users are able to submit the survey more than once, so there could be more than one submission per day.

Also, maybe the chart could be scrollable (?) or able to change the date range to see past data.

Just some ideas to keep in mind.